.github/workflows/deploy_preview_forks.yml

```diff
@@ -0,0 +1,23 @@
+name: Deploy (fork)
+on:
+  pull_request_target:
```

Comment:
only runs on forks

Reply:
This is helpful context. Could you add this as a code comment to the yml file?